presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -188,6 +188,83 @@ public void testReadInsertOnly(boolean isPartitioned, BucketingType bucketingTyp
         }
     }
 
+    @Test(groups = {STORAGE_FORMATS, HIVE_TRANSACTIONAL}, dataProvider = "partitioningAndBucketingTypeDataProvider", timeOut = TEST_TIMEOUT)
```

Comment:
`partitioningAndBucketingTypeDataProvider` does not cover bucketing v1

Reply:
added